plugins/inputs/jolokia2_proxy/README.md

```diff
@@ -81,13 +58,13 @@ Optionally, specify TLS options for communicating with proxies:
     # username = ""
     # password = ""
 
-  [[inputs.jolokia2_agent.metric]]
+  [[inputs.jolokia2_proxy.metric]]
     name  = "jvm_runtime"
     mbean = "java.lang:type=Runtime"
     paths = ["Uptime"]
 ```
 
-### Jolokia Metric Configuration
+### Metric Configuration
```

Comment:
This section is the same as the one in jolokia2_agent/README.md. Rather than copy it, let's only have it in the agent readme. Then we can put some text here to point people there for details. `Metric configuration is the same as in the jolokia2_agent plugin. See its [readme](../jolokia2_agent/README.md) for details` or something similar. Then we won't have to worry about keeping the two files in sync or adding a third file they both link to.